content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -1,22 +1,24 @@
 ---
-title: Managing Resources for Containers
+title: Resource Management for Pods and Containers
 content_type: concept
 weight: 40
 feature:
   title: Automatic bin packing
   description: >
-    Automatically places containers based on their resource requirements and other constraints, while not sacrificing availability. Mix critical and best-effort workloads in order to drive up utilization and save even more resources.
+    Automatically places containers based on their resource requirements and other constraints, while not sacrificing availability.
```

Comment:
```suggestion
    Automatically places containers based on their resource requirements and other constraints without sacrificing availability.
```

Reply:
I hope that's a nit - the phrasing was also wrong in the original.

Reply:
As this is a change to the front page (https://kubernetes.io/) I'd prefer to save wording tweaks here to happen in a separate PR.